examples/hybrid_jobs/2_Using_PennyLane_with_Braket_Jobs/Using_PennyLane_with_Braket_Jobs.ipynb

```diff
@@ -107,14 +107,25 @@
     "nx.write_adjlist(graph, input_file_path)"
```

Comment:
<p>1) Make sure we are saying Braket Hybrid Jobs (for all instances/changes here - generally) </p><p> </p><p>2) comes -&gt; come</p><p> </p><p>3) Lets plz explain what these container environments mean. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
When these notebooks were created, we were using "Amazon Braket Hybrid Jobs" or "Braket Jobs". Maybe this has changed. But I suggest we make this change in a different PR, as it's a change to all notebooks mentioning Jobs in the example repo. This PR is about update for parametric compilation.

Reply:
Container has been mentioned in the getting started with Job notebook. There is also explanation in the developer guide. We made decision that we don't put extensive explanation on what a container is in this notebook.